### ISSUE TITLE
Email character size restriction?

### ISSUE THREAD

**Issue by Organizer21** [tin-cat/emailqueue#12]
Before I debug this further, is there a character limit or something for the email content? It seems my emails are cut short (just stops mid-sentence) on the content side after some 62k+ characters through using the inject class.

**Comment by tin-cat** [comment]
Hi @Organizer21, indeed there is a hard-coded limit on emailqueue_inject.class.php, lines 62-64 that limits the size of the email body to roughly 63Kb. In the past, I made lots of testing experiments with EmailQueue to improve inbox placement and prevent sent emails to be detected as SPAM. SPAM filters are so sensible that there are many of tiny things that can trigger an alarm in a SPAM detection software or SMTP server. It seemed that emails longer than 64Kb. were easily labelled as spam, so I hardcoded this limit (to 63Kb. instead of 64Kb. just to be sure).

To remove this limit, just remove the lines mentioned above!

I guess a more elegant solution would be to make this configurable and let the user choose.

Thanks for the report!
